.github/workflows/validate.yml

```diff
@@ -21,8 +21,8 @@ jobs:
         uses: actions/setup-java@v2
         with:
           distribution: 'zulu'
-          java-version: 17
-          cache: 'gradle'
+          java-version: 20
+      - uses: gradle/gradle-build-action@v2
```

Comment:
From my experience, the official `gradle-build-action` from Gradle do the job better than what `setup-java` does. + it also comes with extra summary that simplifies reading build data from build scans:

![image](https://github.com/airbnb/lottie-android/assets/36954793/f8ef74ed-c2b3-4df8-941c-9152fdd451c4)

the improvement won't be immediately visible, since it works in read-only mode when invokend from PR, but things will start working normally after it's ivoken from the main branch for the first time

Reply:
Neat! Let's give it a shot 😄

Reply:
Thanks! Just to be clear, builds scans (the last colump) won't yet be published, the action is only capable of extracting them from the build output. 
I will configure build scans separately, I planned to do that when proposing changes to the CI pipeline.